coverage-checker.php

```diff
@@ -11,14 +11,10 @@
 }
 
 $xml = new SimpleXMLElement(file_get_contents($inputFile));
-$metrics = $xml->xpath('//metrics');
-$totalElements = 0;
-$checkedElements = 0;
+$metrics = $xml->xpath('/coverage/project/metrics');
```

Comment:
This is because you were counting the metrics 3x - for each class, each file and in the summary. Now it counts only summary.